src/parquet/parquet.h

```diff
@@ -179,6 +190,8 @@ class ColumnReader {
   std::vector<uint8_t> values_buffer_;
   int num_decoded_values_;
   int buffered_values_offset_;
+  //support for fixed length byte arrays
+  int fixed_length_size;
```

Comment:
You can grab this from the `parquet::SchemaElement` below and avoid data type-specific members